apps/src/code-studio/components/pairing/Pairing.jsx

```diff
@@ -171,16 +192,29 @@ export default class Pairing extends React.Component {
           </div>
         ))}
         <div className="clear" />
-        <button type="button" className="stop" onClick={this.handleStop}>
+        <button
+          type="button"
+          className="stop"
+          onClick={this.handleStop}
+          style={{marginLeft: 0}}
```

Comment:
nit: our [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md) prefers using a styles object instead of inline styles. [example](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/BubbleChoice.jsx#L11)